website/src/home/content/about.md

```diff
@@ -0,0 +1,15 @@
+# About
+
+Compa is a companion application for students to manage and access resources at their higher education institution.
+
+As a fresher, settling into school could be easier. You usually need to find:
+
+- Communities/Clubs you are interested in
+- Semester timetables
+- Resources from past semesters
+
+For continuing students, it's just as hard to find documents or class notes when conversations are scattered across multiple platforms.
+
+Being a student is hard enough, we know. That's why Compa aims to be the go-to resource for higher education institutions and save you from one source of stress.
+
+Your open, compact, companion and compass. That is _Compa_.
```

Comment:
```suggestion
Your open compact companion and compass. That is _Compa_.
```

Reply:
the commas are necessary, because grammar. it's also consistent with a similar line in the main README.d

Reply:
hmm. let me think about it a different way. I'll still define compa with the adjectives, but maybe add a one-liner showing what it means?

Compa is:
_Open_: All content is crowdsourced, contributed by people like you and I!
_Compact_: Find all you need for college/university, in one place!
_(Your)Companion_: It exists to help you!
_(Your)Compass_: (no idea)

you get me?

- whatever change we make here may need to be replicated in the

Reply:
Yes, but those are not familiar use of those words. Example, use them as standalone:

Your open.

Your compact.

vs.

Your open compact companion. (Much like, you tall fair friend)

Your compass.

And yes, it should reflect in the main README.